test/fprime_gds/common/loaders/resources/RefTopologyDictionary.json

```diff
@@ -8,6 +8,86 @@
     "dictionarySpecVersion" : "1.0.0"
   },
   "typeDefinitions" : [
+    {
+      "kind" : "alias",
+      "qualifiedName" : "FwTlmPacketizeIdType",
```

Comment:
How come only these types are in the JSON dictionary, when it seems there's a lot more in `FpConfig.fpp` ?

Reply:
For the JSON dictionary, I only require the types that are needed by the GDS. Currently, the GDS only uses (and allows the user to override) the following types: https://github.com/nasa/fprime-gds/blob/devel/src/fprime_gds/common/utils/config_manager.py#L154-L162

Reply:
We could expand the types that are required by the dictionary, but they would not be used in the GDS.

Reply:
Okok, I guess this is just a test dictionary – but I would expect the actual dictionary generated by `fprime/Ref/` to include all type aliases, and not only the ones needed for the GDS, is that correct?

Reply:
Nope! FPP only requires the the following types when building the dictionary:
- FwChanIdType
- FwEventIdType 
- FwOpcodeType
- FwPacketDescriptorType
- FwTlmPacketizeIdType

Reason we went with this set is because they are the types that are configurable by FPrime and the GDS (and the GDS needs to know what they are in order to decode things, etc.).

Reply:
Do you mean to say that anything used in Commands/Events/Telemetry will be included because they are used as part of other dictionary objects and that the above set is the core set of types that describe the packets themselves and aren't strictly contained within the model of an element (e.g. command) within those packets?

What if the user defines another packet protocol (e.g. CCSDS) and needs to have FPP constants that influence Flight and Ground but that does not end up a modeled type (e.g. command)?

Reply:
Apart from the extra amount of data stored in the dictionary, why wouldn't we include all non-opaque types (modeled types) in the dictionary?  If we know the type's definition, why not provide it?

Reply:
> Do you mean to say that anything used in Commands/Events/Telemetry will be included because they are used as part of other dictionary objects and that the above set is the core set of types that describe the packets themselves and aren't strictly contained within the model of an element (e.g. command) within those packets?

Yes, that sounds right!

> What if the user defines another packet protocol (e.g. CCSDS) and needs to have FPP constants that influence Flight and Ground but that does not end up a modeled type (e.g. command)?

In this implementation, for the example you provided (a user defines another packet protocol and defines some FPP constant), the type would _not_ be included in the dictionary since it does not appear in some command/event/etc.